app/src/main/java/ch/nevis/exampleapp/coroutines/ui/transactionConfirmation/parameter/TransactionConfirmationNavigationParameter.kt

```diff
@@ -16,27 +15,21 @@ import kotlinx.parcelize.Parcelize
  * Navigation parameter class for Transaction Confirmation view.
  *
  * @constructor Creates a new instance.
- * @param operation The operation the account selection was requested for.
- * @param accounts The list of available accounts the user can select from.
- * @param transactionConfirmationData The transaction confirmation data/message that should be displayed
- *  on Transaction Confirmation view.
+ * @param account The previously selected account.
+ * @param transactionConfirmationMessage The transaction confirmation data/message that should be displayed
+ * on Transaction Confirmation view.
  */
 @Parcelize
 data class TransactionConfirmationNavigationParameter(
 
     /**
-     * The operation type the account was selection requested for.
-     */
-    val operation: Operation,
```

Comment:
❓: Just to clarify it. We can get rid of `operation` property because it will be always out-of-band authentication?

Reply:
Yes, that's what I thought, there was even a comment saying it's always that so I thought what is the reason to pass it around then? Or do you have a reason for it?